actors/builtin/power/policy.go

```diff
@@ -4,11 +4,22 @@ import (
 	abi "github.com/filecoin-project/specs-actors/actors/abi"
 )
 
-// Minimum number of registered miners for the minimum miner size limit to effectively limit consensus power.
-const ConsensusMinerMinMiners = 3
+// PARAM_SPEC
+// This is a switch for turning on the minimum miner size requirement.
+// When this number of miners of minimum miner size is reached, minimum miner size is enforce.
+const ConsensusMinerMinMiners = 4
 
-// Minimum power of an individual miner to meet the threshold for leader election.
-var ConsensusMinerMinPower = abi.NewStoragePower(1 << 40) // PARAM_FINISH
+// PARAM_SPEC
+// Minimum miner size, the minimum power of an individual miner to meet the threshold for leader election (in bytes).
+// Motivation:
+// - Limits sybil generation
+// - Improves consensus fault detection
+// - Guarantees a minimum fee for consensus faults
+// - Ensures that a specific soundness for the power table
+// Future: we can consensus fault fee and sybil generation with crypto econ mechanic and we can mantain the target soundness by increasing the challenges for small miners.
+var ConsensusMinerMinPower = abi.NewStoragePower(100 << 40) // PARAM_FINISH
 
+// PARAM_SPEC
 // Maximum number of prove commits a miner can submit in one epoch
+// Motivation: This aims at limiting the number of operations on chain per miner.
 const MaxMinerProveCommitsPerEpoch = 8000
```

Comment:
**Clarification request:** @ZenGround0 and @anorth can you confirm that the motivation is correct?

Reply:
Not really. It's aimed at what the original comment says, because those prove commits must all be processed in end-of-epoch cron.

Reply:
I understand it is aimed at that (I left the original comment), but what is the motivation for choosing 8000?